------------------------------------------------------------
revno: 13234
revision-id: squid3@treenet.co.nz-20160420101437-36eofkldxfku61kj
parent: squid3@treenet.co.nz-20160420063907-hcnf4qmcg5hbjc11
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.4
timestamp: Wed 2016-04-20 22:14:37 +1200
message:
  nullptr is a C++11 feature
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20160420101437-36eofkldxfku61kj
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.4
# testament_sha1: 0f9f2a5b29a7ef02befe2a4e2e6357ef4bcffbce
# timestamp: 2016-04-20 10:51:03 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.4
# base_revision_id: squid3@treenet.co.nz-20160420063907-\
#   hcnf4qmcg5hbjc11
# 
# Begin patch
=== modified file 'src/tests/stub_cbdata.cc'
--- src/tests/stub_cbdata.cc	2016-04-20 06:39:07 +0000
+++ src/tests/stub_cbdata.cc	2016-04-20 10:14:37 +0000
@@ -10,7 +10,7 @@
 }
 void *cbdataInternalFree(void *p, const char *, int) {
     xfree(p);
-    return nullptr;
+    return NULL;
 }
 #if USE_CBDATA_DEBUG
 void *cbdataInternalAllocDbg(cbdata_type type, const char *, int) STUB_RETVAL(NULL)

